.github/workflows/nodejs.yml

```diff
@@ -1,12 +1,16 @@
 name: Node.js CI
 
-on: [push,pull_request]
+on:
```

Comment:
You can simplify this down to just `on: [push,pull_request]` unless non-master patches happen a lot?

Reply:
In that case it runs both 'push' and 'pull_request' if the development branch is in upstream and not in the fork as it is the case with this one. We usually don't do it, this time it happened cause I've used GitHub UI to create the actions file, but being explicit wouldn't hurt imho